packages/quantic/cypress/e2e/default-2/document-suggestion/document-suggestion.cypress.ts

```diff
@@ -91,29 +293,13 @@ describe('quantic-document-suggestion', () => {
         Expect.numberOfSuggestions(defaultMaxDocuments);
         Expect.displayAccordionSectionContent(true, 0);
       });
-
-      scope('when clicking on a document suggestion', () => {
-        const clickIndex = 1;
-
-        Actions.clickSuggestion(clickIndex);
-        Expect.logClickingSuggestion(clickIndex, allDocuments);
-        Expect.displayAccordionSectionContent(true, 0);
-        Expect.displayAccordionSectionContent(true, clickIndex);
-      });
-
-      scope('when rating a document suggestion', () => {
-        const clickIndex = 1;
-
-        sendRating(clickIndex);
-        Expect.logRatingSuggestion(clickIndex, allDocuments);
-      });
     });
   });
 
   describe('when using a custom number maxDocuments', () => {
-    const maxDocuments = 3;
+    const maxDocuments = 2;
```

Comment:
?

Reply:
3 was the default number of max documents, so I'm using a real custom value in this test, 2.